### DISCUSSION TITLE
'View' and 'Add' group privileges but without Edit- possible?

### DISCUSSION THREAD

**Discussion by wafflecrate** [inventree/InvenTree#4858] [Q&A]
I created an Inventree account for a 2nd user. I set it up so [he has 'Staff Status' ](https://imgur.com/9OzQfms) so he can log in to our Inventree account with his own account. I also added him to a Group called "Production Workers" and I want to set it up so people in this group can only:

- View Parts
- Count Part Stock
- Transfer Part Stock

I do not want Users in this group to be able to go into the ["Edit Part" section for each Part.](https://imgur.com/6shkJgk)

Is such a thing possible? Because I tried to edit the Group permissions so they only have "View" and "Add" privileges but after hitting Save I noticed that this page automatically [forces the 3rd "Change" option](https://imgur.com/cd8P7sv). So it basically forces V | A | C and doesn't allow only V | A. Is it possible to Have V | A or does it automatically switch on the C?

**Comment by SchrodingersGat** [comment]
If you have "add" permission for any particular [role](https://docs.inventree.org/en/latest/settings/permissions/#roles) then the "change" permission is automatically added in too. This does not *have* to be the case but it made sense to us when we were implementing it. Making such a feature available would require a code change, unless you can have a different way of working around the issue

**Reply by wafflecrate** [reply]
Are  'Edit Part' entries logged in a viewable History log somewhere in the app? That could be a potential workaround. As long as I can see which employee made an Edit to a parts item and roll back the changes they might have made.

**Reply by matmair** [reply]
There is a proposal for this but nothing is implemented.
